bot/connector-google-chat/src/test/kotlin/GoogleChatFootnoteFormatterTest.kt

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (C) 2017/2025 SNCF Connect & Tech
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import ai.tock.bot.connector.googlechat.GoogleChatFootnoteFormatter
+import ai.tock.bot.engine.action.Footnote
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class GoogleChatFootnoteFormatterTest {
+
+    @Test
+    fun `format with no footnotes returns original text`() {
```

Comment:
You can improve/refactor this overall test file, using a list of test cases though ParameterizedTest see an exemple here : https://proandroiddev.com/kotlin-unit-tests-with-parameters-e37aab2b36f6